src/d3a/d3a_core/export.py

```diff
@@ -93,6 +93,9 @@ def export_json_data(self, directory: dir):
         trade_file = os.path.join(json_dir, "trade-detail")
         with open(trade_file, 'w') as outfile:
             json.dump(self.endpoint_buffer.trade_details, outfile, indent=2)
+        device_file = os.path.join(json_dir, 'device_statistics')
```

Comment:
I guess these three line could be removed as https://github.com/gridsingularity/d3a/pull/378/files#diff-9e95bed0f62db1b1f967c65f0823d49dR100 it is taking care of dumping device statistics as json

Reply:
You are right! Thanks!